spyder/plugins/findinfiles/widgets/main_widget.py

```diff
@@ -511,10 +518,7 @@ def _get_options(self):
         if text_re:
             error_msg = regexp_error_msg(texts[0][0])
             if error_msg:
-                self.search_text_edit.lineEdit().setStyleSheet(
-                    self.REGEX_INVALID)
-                tooltip = self.REGEX_ERROR + ': ' + str(error_msg)
-                self.search_text_edit.setToolTip(tooltip)
+                self.show_error(str(error_msg), False)
```

Comment:
```suggestion
                self._show_error(str(error_msg), False)
```

Make method private.